apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-utils.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Block Patterns Utils.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Class Block_Patterns_Utils
+ */
+class Block_Patterns_Utils {
+	/**
+	 * Make remote get requests.
+	 *
+	 * @param string $request_url The request URL.
+	 * @return array              The response.
+	 */
+	public function remote_get( $request_url ) {
+		$args = array( 'timeout' => 20 );
+
+		if ( function_exists( 'wpcom_json_api_get' ) ) {
+			$response = wpcom_json_api_get( $request_url, $args );
+		} else {
+			$response = wp_remote_get( $request_url, $args );
+		}
+
+		if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
+			return array();
+		}
+		return json_decode( wp_remote_retrieve_body( $response ), true );
+	}
+
+	/**
+	 * A wrapper for wp_cache_add.
+	 *
+	 * @param  int|string $key    The cache key to use for retrieval later.
+	 * @param  mixed      $data   The data to add to the cache.
+	 * @param  string     $group  Optional. The group to add the cache to. Enables the same key to be used across groups. Default empty.
+	 * @param  int        $expire Optional. When the cache data should expire, in seconds.
+	 *                            Default 0 (no expiration).
+	 * @return bool               True on success, false if cache key and group already exist.
+	 */
+	public function cache_add( $key, $data, $group, $expire ) {
```

Comment:
Just noticed that this signature makes the `$group` and `$expire` params required params, which is probably fine in our context (for block patterns we don't want to call this without a group or an expiration). Shall we update the doc comment to remove `Optional`? (Or do you want to match the behaviour in `wp_cache_add`)?

Reply:
👍